warehouse/pom.xml

```diff
@@ -232,7 +232,7 @@
             <dependency>
                 <groupId>org.apache.commons</groupId>
                 <artifactId>commons-lang3</artifactId>
-                <version>3.3.2</version>
+                <version>3.11</version>
```

Comment:
Was this change hygiene or to leverage functionality in the newer version?

Reply:
It was to get access to the newer Failable* classes. I used FailableSupplier to handle the possible exception in the lambda